### DISCUSSION TITLE
Welcome to community-feedback Discussions!

### DISCUSSION THREAD

**Discussion by hohonuuli** [fathomnet/community-feedback#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello everyone! I'm Dan—I'm a 2nd year community college student from San Diego looking to transfer to UCSD / Scripps Oceanography next fall for physical oceanography. My dream goal is a PhD in Applied Ocean Science from SIO. I was originally a film student who wanted to be a cinematographer, but when the pandemic hit I decided to do something completely out of my comfort zone: math has always been my weakest subject so I figured, "Why not be an oceanographer or an engineer?" The story is much longer than that, but maybe for another time.

My first exposure to ROV exploration was in October 2020, when I stumbled upon Schmidt Ocean Institute's YouTube. It was then that I knew what I wanted to be involved with in the future. I love the science, but I especially love the use of camera technology, imaging, and telepresence as it relates to deep sea exploration and marine ecology—not only for the much needed research but also because it can communicate stories while inspiring and engaging others. Seeing as how FathomNet combines imagery, data, and community involvement, I'm just really excited about it and I'm glad to be here!
